cmd/beacon-chain/flags/base.go

```diff
@@ -23,6 +23,13 @@ var (
 		Usage: "An http endpoint for an Ethereum execution node",
 		Value: "",
 	}
+	// ExecutionJWTSecretFlag provides a path to a file containing a hex-encoded, 256 bit JWT secret
+	// which will be used to authenticate with Ethereum execution nodes via an HTTP connection.
+	ExecutionJWTSecretFlag = &cli.StringFlag{
+		Name:  "jwt-secret",
+		Usage: "File path to a 256 bit, hex-encoded JWT secret used for authenticating with Ethereum execution nodes via HTTP",
```

Comment:
Can you add more to the usage? Perhaps elaborating on some of the default behavior or error causes would be helpful for the user.

https://github.com/ethereum/execution-apis/blob/main/src/engine/authentication.md#key-distribution

> If such a parameter is not given, the client SHOULD generate such a token, valid for the duration of the execution, and store it the hex-encoded secret as a jwt.hex file on the filesystem. This file can then be used to provision the counterpart client.
>
> If such a parameter is given, but the file cannot be read, or does not contain a hex-encoded key of at least 256 bits, the client should treat this as an error: either abort the startup, or show error and continue without exposing the authenticated port.